src/coreclr/gc/objecthandle.h

```diff
@@ -71,6 +71,10 @@ uint32_t     CompareExchangeVariableHandleType(OBJECTHANDLE handle, uint32_t old
  */
 
 int GetCurrentThreadHomeHeapNumber();
+gc_alloc_context* GetCurrentThreadAllocContext();
+
+// helper for getting the total number of processors
```

Comment:
just a documentation nit here - is it intended that this will always be number of processors, or could it change in the future?  (in other words, what's the abstraction here - slots or processors - if the former, is the comment over-specifying the behavior, or if the latter, should the function be renamed?)

Reply:
good point - I will revise this to say it's the # of table slots (which for server GC happens to be the # of processors) we allocate in